blocks/color-palette/index.js

```diff
@@ -23,7 +23,7 @@ export function ColorPalette( { colors, disableCustomColors = false, value, onCh
 
 	return (
 		<div className="blocks-color-palette">
-			{ map( colors, ( color ) => {
+			{ map( colors, ( { color } ) => {
```

Comment:
In my console:

```
Warning: Each child in an array or iterator should have a unique "key" prop.
```

**Edit:** Actually, this is because I had https://github.com/WordPress/gutenberg/pull/5570#pullrequestreview-106634138 in place. Still an issue, though not certain it's introduced here. No warning when many color options exist.